mj_envs/mujoco_env.py

```diff
@@ -81,7 +81,15 @@ def viewer_setup(self):
         """
         pass
 
+    def get_proprioception(self, use_tactile):
+        """
+        For the VIL paper
+        """
+        pass
+
     # -----------------------------
+    def get_body_com(self, body_name):
```

Comment:
These are environment specific utilities that you can just add to the env definition. Im not sure if there is any benefit of adding this to the mujoco_env base class. 

@aravindr93 - what do you think?

Reply:
I am not sure if this function is correct in the first place. This is returning the xpos of the body, which is the root position of the body in global coordinates, but need not be center of mass. @vikashplus, is this correct? Ideally, you would have to look at all the members of this body, and do a weighted average of their xpos.

If this function is critical, and we are actually using it (meaning you need the COM and not just the root position), then it makes sense to have it in mujoco_env.py, since it might be useful for many envs and future applications.

Reply:
Here is how you can get the COM
    mjtNum*   xipos;                // Cartesian position of body com           (nbody x 3)

Seems like get_body_xpos is giving you the location of the root of the body, not necessarily the body_com

Reply:
I think we can remove this function for now.